packages/control/index.d.ts

```diff
@@ -24,7 +24,7 @@ declare namespace control {
     platformaticVersion: string
   }
 
-  interface Services {
+  interface Service {
```

Comment:
Minor change to keep it consistent with the other interfaces since they all use singular naming.
This should not be used on the "outside," so it should be a safe change.

Reply:
This name is incorrect. This should be something like “RuntimeConfig” or “RuntimeData” or “RuntimeServices”. A service is contained by this in the services property.